lib/tabula.js

```diff
@@ -126,10 +126,8 @@ function tabulaFormat(items, options) {
     assert.optionalArrayOfString(options.sort, 'options.sort');
     assert.optionalArrayOfString(options.validFields, 'options.validFields');
     assert.optionalBool(options.dottedLookup, 'options.dottedLookup');
+    assert.ok(options.columns || items[0], 'column names cannot be determined');
 
-    if (items.length === 0) {
-        return '';
-    }
     var cols = [];
```

Comment:
doesn't this mean a current user upgrading could now be hitting an assert for some data case where they don't have any rows?  Hrm.   I wonder if some behaviour rather than assert here. Thoughts?